### ISSUE TITLE
RFC Ameldabor official repo

### ISSUE THREAD

**Issue by piejanssens** [Amelandbor/service.subtitles.addic7ed#5]
Looks like you're doing a good job in maintaining the addic7ed addon.
As you probably know at one point addic7ed did not want Kodi (or xbmc at that time) to have an addic7ed addon in the official repo. So it was removed. 
Now a while ago one of the addic7ed admins who goes by the name alex28 tried to reach out on facebook to someone from Kodi to bring back the addic7ed support. https://www.facebook.com/Addic7ed/posts/10152734938466723

Are you willing to become the maintainer of the official Kodi addic7edaddon? If so, could you get in touch with alex28 to get his permission to put your fork on the Kodi repo?

**Comment by Amelandbor** [comment]
Hi @piejanssens,

I've tried to contact alex28 on IRC with no success. And I have no facebook, so it's a bit hard to reach him.

**Comment by piejanssens** [comment]
I just tried to get in touch with him on Twitter and Facebook. I'll let you know as soon as I hear something from him. If it says quiet, would you be willing to host this addon on your own repository? You can use github itself as a Kodi repo. Much like it's done here https://github.com/teeedubb/teeedubb-xbmc-repo As you can see there is a script that can generate the required addons.xml for you.

**Comment by Amelandbor** [comment]
If necessary that's possible.

**Comment by disrupted** [comment]
@Amelandbor @piejanssens auto updates through a Kodi repo would be very much appreciated :)

**Comment by cudencuden** [comment]
any update??

**Comment by disrupted** [comment]
no news ? the addon is throwing errors at me lately @Amelandbor @piejanssens

**Comment by Grossdm** [comment]
@freeman212
Hello,
I also had errors a while back.
Disclosure: I'm not a developer.
After some debug logging, etc. I modified the code slightly. It seems to have worked, as it immediately stopped the errors I was getting and I haven't had any since.

I think most people on Github would do something called a pull request with this, but I've never pulled a request in my life. 😆
Then again, if this code is essentially abandoned, perhaps a fork would be best. This is another thing I've never done.

If you would like my "tweak" to the code I'm happy to share it.
Just let me know how to get it to you (or properly handle it on Github).

Doug

**Comment by Amelandbor** [comment]
Hi @freeman212 and @Grossdm,

There seems to be an issue when the site is busy. I haven't done any proper debugging though.
@Grossdm you can click on a file you want to edit and click the edit button in the top right. It creates a fork from which you can immediately create a PR.

If you can't figure it out you can always show me the edits you've made so I can do it myself.

Cheers,

Amelandbor

**Comment by disrupted** [comment]
@Grossdm @Amelandbor
awesome ! Looking forward to see the changes

**Comment by cudencuden** [comment]
if no timely response is received.. just fork it... plenty of guides on Google on how

**Comment by Amelandbor** [comment]
@freeman212 I've quickly updated the timeout settings to 20 seconds, can you try if that works for you?

**Comment by disrupted** [comment]
@Amelandbor can't do so right now, but will check it later or tomorrow when I am home. thanks so far!

**Comment by larpo** [comment]
Seems to be working fine now!

**Comment by disrupted** [comment]
Can confirm, it's working again although the search is of course a little slow

**Comment by peter099** [comment]
It stopped working for me. Getting 'addicted.com error' when searching for subtitles.

**Comment by Grossdm** [comment]
@peter099, this might be bad form to recommend a different addon, but I have found a different Addic7ed addon that works well for me.

The webpage for the repo is [http://romanvm.github.io/kodi_repo](http://romanvm.github.io/kodi_repo), or here is a [direct link](https://romanvm.github.io/kodi_repo/repo/repository.romanvm/repository.romanvm-2.0.0.zip) to download the repo's zip file.

The dev has other addons in his repo and I have only tried his service.subtitles.addic7ed.

I hope this helps and that I haven't been rude posting this I another dev's repo.

Doug®
